app/src/renderer/vuex/modules/blockchain.js

```diff
@@ -61,6 +81,66 @@ export default ({ commit, node }) => {
         })
       })
     },
+    async queryTxInfo({ state, dispatch }, height) {
+      let blockTxInfo = state.blockTxs.find(
```

Comment:
why not store them per height?

Reply:
was following method used by `queryBlockInfo()` but you're right keying the height is smart i'll update both of em

Reply:
actually just updating the blockchain one for sake of PR scope

Reply:
actually i think both of these need to stay as arrays as they are used in for loops

Reply:
Can you point me to where they are used in loops? What I have found, there is always a selection by height happening when blockTxs appear in code.

Reply:
oops i totally misinterpreted the unit test error relating to `should disable the next block button if last block`. thought that info was counting an array length for looping and failed when it became an obj (as per a similar error in `blockchain.spec.js`). i've gone back and updated to a keyed storage method for `blockTxs` as well as `blockMetas`